certified-connectors/Tesseron ASM Ticket/apiProperties.json

```diff
@@ -0,0 +1,42 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "instanceURL": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Instance URL",
+          "description": "Your Tesseron ASM instance URL",
+          "tooltip": "Provide the Tesseron ASM instance URL (without www.)",
+          "constraints": {
+            "required": "true"
+          }
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "apikey",
+          "description": "Your Tesseron ASM API key (Service: Ticket)",
+          "tooltip": "SProvide your Tesseron ASM API key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#218472",
```

Comment:
@developmentLL make sure that your icon background color is not transparent and it matches the color from this property. I noticed that you have submitted an icon along with other files, it has transparent background. It's not allowed.